style.css

```diff
@@ -0,0 +1,192 @@
+html {
```

Comment:
- [ ] **Required:** Please take some time to review your whole design before your next submission. 
It seems to be broken.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/159551041-5f741d7f-2511-4b01-adc7-0dcfaa146f2d.png) | ![image](https://user-images.githubusercontent.com/27709832/159551383-a6146448-6db7-47ec-b9b9-2ee9b2a2c05f.png)

Reply:
check now  is i okey??

Reply:
- [ ] Kindly work on your design as required by the previous reviewer. You can reference the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) given in the project brief and style your app in the same look and feel. 👍